pkgs/development/libraries/getdns/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchurl, libtool, unbound, libidn, m4, file
+, openssl, doxygen, autoreconfHook, automake, check, libbsd }:
+
+stdenv.mkDerivation rec {
+  pname = "getdns";
+  name = "${pname}-${version}";
+  version = "1.3.0";
+
+  src = fetchurl {
+    url = "https://getdnsapi.net/releases/${pname}-1-3-0/${pname}-${version}.tar.gz";
+    sha256 = "920fa2e07c72fd0e5854db1820fa777108009fc5cb702f9aa5155ef58b12adb1";
+  };
+
+  nativeBuildInputs = [ libtool m4 libbsd ];
+
+  buildInputs = [ unbound libidn openssl doxygen autoreconfHook
+   automake file check ];
+
+  patchPhase = ''substituteInPlace m4/acx_openssl.m4 \
+    --replace /usr/local/ssl ${openssl.dev}
+    '';
+
+  meta = with stdenv.lib; {
+    description = "A modern asynchronous DNS API";
+    longDescription = ''
+getdns is an implementation of a modern asynchronous DNS API; the
+specification was originally edited by Paul Hoffman. It is intended to
+make all types of DNS information easily available to application
+developers and non-DNS experts. DNSSEC offers a unique global
+infrastructure for establishing and enhancing cryptographic trust
+relations. With the development of this API the developers intend to
+offer application developers a modern and flexible interface that
+enables end-to-end trust in the DNS architecture, and which will
+inspire application developers to implement innovative security
+solutions in their applications.
+'';
+    homepage = "https://getdnsapi.net";
```

Comment:
Use
```
homepage = https://getdnsapi.net;
```

(URL is a first-class type in Nix.)

Reply:
Done.